contracts/staking.tmy/include/staking.tmy/staking.tmy.hpp

```diff
@@ -67,7 +59,7 @@ namespace stakingtoken
         // TODO: make sure that this does not need to be rounded otherwise will create a bug. if the cycle is 17 seconds and the period is 3 seconds there. 3 does not fit into 17 exactly
```

Comment:
L47: can you also change static const `uint8_t MAX_ALLOCATIONS = 20;` because I'm a bit worried that many will overload the cron job